frontend/app/src/composables/dark-mode.ts

```diff
@@ -1,13 +1,56 @@
-export const useDarkMode = () => {
+import { ThemeMode, useRotkiTheme } from '@rotki/ui-library-compat';
+
+export const useDarkMode = createSharedComposable(() => {
+  const { config, switchThemeScheme, setThemeConfig } = useRotkiTheme();
+  const { theme } = useTheme();
   const store = useSessionStore();
   const { darkModeEnabled } = storeToRefs(store);
 
   const updateDarkMode = (enabled: boolean) => {
     set(darkModeEnabled, enabled);
   };
 
+  watchDeep(
+    theme,
+    theme => {
+      // @ts-ignore
```

Comment:
what are we ignoring here? :)

Reply:
I need access to `parsedTheme`, but it is non enumerable, and not defined in the type